server/services/store/mattermostauthlayer/mattermostauthlayer.go

```diff
@@ -38,6 +40,16 @@ func New(dbType, connectionString string, store store.Store) (*MattermostAuthLay
 
 		return nil, err
 	}
+	maxDBIdleConns, err := strconv.Atoi(os.Getenv("FOCALBOARD_DB_MAX_IDLE_CONNS"))
+	if err != nil {
+		maxDBIdleConns = 0
+	}
+	maxDBOpenConns, err := strconv.Atoi(os.Getenv("FOCALBOARD_DB_MAX_OPEN_CONNS"))
+	if err != nil {
+		maxDBIdleConns = 0
```

Comment:
This should set `maxDBOpenConns`. Can we use a reasonable default instead of 0 (unlimited)? Per #616, suggest the following defaults:
* MaxOpenConns: 300
* MaxIdleConns: 20
* ConnMaxLifetimeMilliseconds: 3600000 (60 minutes)
* ConnMaxIdleTimeMilliseconds: 300000 (5 minutes)

Reply:
:+1: